.github/workflows/continuous-integration.yml

```diff
@@ -47,20 +47,16 @@ jobs:
           restore-keys: ${{ runner.os }}-composer-
 
       - name: Install dependencies
-        if: ${{ matrix.php-versions != '8.0' }} # Remove this as soon as `drupol/php-conventions` supports PHP 8
         run: composer install --no-progress --no-suggest --prefer-dist --optimize-autoloader
 
       - name: Run Grumphp
-        if: ${{ matrix.php-versions != '8.0' }} # Remove this as soon as `drupol/php-conventions` supports PHP 8
         run: vendor/bin/grumphp run --no-ansi -n
 
       - name: Send PSALM data
-        if: ${{ matrix.php-versions != '8.0' }} # Remove this as soon as `drupol/php-conventions` supports PHP 8
         run: vendor/bin/psalm --shepherd --stats
```

Comment:
@drupol There is an error in the `Send PSALM data` step, I don't know if this should work...
```
  vendor/bin/psalm --shepherd --stats
  shell: /usr/bin/bash -e {0}
/home/runner/work/_temp/2a3a4068-c3b0-4825-ab78-2293ac5a2797.sh: line 1: vendor/bin/psalm: No such file or directory
Error: Process completed with exit code 127.
```

Reply:
yes, please update with: https://github.com/loophp/collection/blob/master/.github/workflows/static-analysis.yml#L60

And don't forget to add `vimeo/psalm` in `composer.json`